packages/hoppscotch-common/src/composables/codemirror.ts

```diff
@@ -324,7 +327,8 @@ export function useCodemirror(
       EditorView.domEventHandlers({
         scroll(event) {
           if (event.target && options.contextMenuEnabled) {
-            handleTextSelection()
+            // Debounce to make the performance better
+            debouncedTextSelection(30)()
```

Comment:
Do we need a similar optimization here?

https://github.com/hoppscotch/hoppscotch/blob/faab1d20fde9a6be660db40fc73dcf28f9038008/packages/hoppscotch-common/src/components/smart/EnvInput.vue#L467-L469

Reply:
yup missed this